README.md

```diff
@@ -34,6 +34,41 @@ Optionally, installation can forced with the 'force' parameter.
 cpan_module 'App::Munchies'
   force true
 end
+```
+
+You can also use [cpanm's version mechanism](http://search.cpan.org/~miyagawa/App-cpanminus-1.7027/bin/cpanm#COMMANDS) to grab a specific version, or glob a version.
+
+Exactly version 1.01 of `App::Munchies` will be installed:
+
+```ruby
+cpan_module 'App::Munchies'
+  version '== 1.01'
+end
+```
+
+At least version 1.01 of `App::Munchies` will be installed:
+
+```ruby
+cpan_module 'App::Munchies'
+  version '1.01'
+end
+```
+
+At least version 1.01 will be installed, but not version 2:
+
+```ruby
+cpan_module 'App::Munchies'
+  version '>= 1.01, < 2.0'
+end
+```
+
+Additionally, you can use the `cpan_module` LWRP to delete a given package (uses cpanm's `--uninstall` param)
+
+```ruby
+cpan_module 'App::Munchies'
+  action :delete
```

Comment:
I'm not sure if delete is the best name of the action.  Uninstall seems to be the native name with cpan unless I'm mistaken. Seems like it would make more sense if this was action uninstall.